lib/chef/knife/windows_cert_generate.rb

```diff
@@ -28,15 +28,19 @@ class WindowsCertGenerate < Knife
 
       banner "knife windows cert generate (options)"
 
-      option :domain,
-        :short => "-d DOMAIN",
-        :long => "--domain DOMAIN",
-        :description => "By default there will be no domain name. If user wants to give the hostname as '*.mydomain.com' then he must specify the domain as: --domain 'mydomain.com'"
+      option :hostname,
+        :short => "-H HOSTNAME",
+        :long => "--hostname HOSTNAME",
+        :description => "By default hostname is *. If user wants to give his fqdn as the hostname
+        then he must specify the hostname as: --hostname 'something.mydomain.com'. User can also give wildcard
+        as '*.mydomain.com'"
 
       option :output_file,
         :short => "-o PATH",
         :long => "--output-file PATH",
-        :description => "By default 3 files will be generated in the current folder as winrmcert.pfx and winrmcert.pem. You can specify alternate file path using this option. Eg: --output-file /home/.winrm/server_cert.pfx. This will create 3 files in the specified path with name server_cert.pem, sever_cert.pfx, server_cert.der.",
+        :description => "By default 3 files will be generated in the current folder as winrmcert.pfx and
+        winrmcert.pem. You can specify alternate file path using this option. Eg: --output-file /home/.winrm/server_cert.pfx.
+        This will create 3 files in the specified path with name server_cert.pem, sever_cert.pfx, server_cert.der.",
```

Comment:
We just verified that adding extension `pfx` in `output_file` is misleading. So `--output-file /home/.winrm/server_cert` will just create 3 files at path '/home/.winrm' with name 'server_cert'.
Updating the description accordingly.